frontend/src/components/jobs/table/JobsTable.jsx

```diff
@@ -43,8 +43,11 @@ export default function JobsTable() {
   // consume zustand store
   const { range, fromTimeIsoStr, onTimeIntervalChange } = useTimePickerStore();
 
+  // state
+  const [initialLoading, setInitialLoading] = React.useState(true);
+
   // API/ Table
-  const [data, tableNode, refetch] = useDataTable(
+  const [data, tableNode, refetch, , loadingTable] = useDataTable(
```

Comment:
The empty space is not wrong and work as intended (ignore param during destructuring), but I suggest to put a `_` I think is more clean.
`const [data, tableNode, refetch, , loadingTable] = useDataTable(` replace with `const [data, tableNode, refetch, _, loadingTable] = useDataTable(`